edgexfoundry/edgex-compose
# build: Update app service to 3.0.1 patch version

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run `make run as-llrp`
Verify ASC and App LLRP images are version 3.0.1 
Verify the /version, /config and /secret endpoint require JWT token
Verify the /ping endpoint does not require JWT token
